ducktape/services/service.py

```diff
@@ -69,23 +68,58 @@ def __repr__(self):
                              (self.num_nodes, [n.account.hostname for n in self.nodes]))
 
     @property
+    def name(self):
+        """Human-readable identifier (almost certainly) unique within a test run."""
+        return self.__class__.__name__ + "-" + str(self.id)
```

Comment:
Is it worth adding a way to override this, for example via constructor parameter? Seem like `self.id` is not going to be all that useful, although I guess in a lot of our current cases we only have one service of each type (and that is something of a constraint of ducktape's current model for mapping services to cluster nodes).